util/find-doc-nits

```diff
@@ -884,6 +894,19 @@ if ( $opt_c ) {
 
 # Preparation for some options, populate %name_map and %link_map
 if ( $opt_l || $opt_u || $opt_v ) {
+    if ( $SRCDIR ne $BLDDIR ) {
+        # If building out-of-tree, foo.pod.in in $SRCDIR should be found as
+        # foo.pod in $BLDDIR;
+        foreach my $in ( glob('doc/*/*.pod.in doc/internal/*/*.pod.in') ) {
+            my $pod = "$BLDDIR/$in";
+            $pod =~ s/\.in$//;
+            if ( -f $pod ) {
+                collectnames($pod);
+            } else {
+                err("$in exists but $pod does not exist")
+            }
+        }
+    }
     foreach ( glob('doc/*/*.pod doc/internal/*/*.pod') ) {
         collectnames($_);
     }
```

Comment:
Why a second loop? ...  Oh, now I see the `$SRCDIR ne $BLDDIR` check, but why on earth?

Reply:
I do not understand your question.